likes_lookup.go

```diff
@@ -0,0 +1,54 @@
+package gotwtr
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+func likesLookUpUsers(ctx context.Context, c *client, id string, opt ...*LikesLookUpUserOpts) (*LikesLookUpUserResponse, error) {
+	// check id
+	if len(id) == 0 {
+		return nil, errors.New("likes look up by tweet: id parameter is required")
```

Comment:
This error message is strange.
```suggestion
		return nil, errors.New("look up likes by users: tweet id parameter is required")
```